README.md

```diff
@@ -12,7 +12,7 @@ defined in the MapML (Map Markup Language)
 
 Beginning with `v0.8.2`, &lt;mapml-viewer&gt; can be loaded from the [UNPKG content delivery network](https://unpkg.com/) –
```

Comment:
`v0.8.2` 🧐

Reply:
Even though I unpublished that version for lacking the `marker-` pngs, it was the first working version available at UNPKG. I'll stand by that statement.

Reply:
Ah, yes.